app/errors.py

```diff
@@ -0,0 +1,16 @@
+class VaccineError(Exception):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
```

Comment:
it's not ok to have one error message for two different errors. Also, `NotVaccinatedError` and `OutdatedVaccineError` are called for one person and it's not logical to write "all friends"